pop-miners/altchain-pop-miner/src/main/kotlin/org/veriblock/lite/NodeCoreLiteKit.kt

```diff
@@ -49,8 +58,16 @@ class NodeCoreLiteKit(
             throw IOException("Unable to create directory")
         }
 
-        this.gateway = NodeCoreGateway(context.networkParameters)
         addressManager = loadAddressManager()
+
+        if (context.networkParameters.isSpv) {
+            this.spvContext = initSpvContest(context.networkParameters, addressManager.all)
```

Comment:
```suggestion
            spvContext = initSpvContest(context.networkParameters, addressManager.all)
```
There is no need to use this in there (same for the rest)

Reply:
it makes more clear, which are class or local variables. But it's just cosmetic things.